website/src/main/tut/docs/users/configuration.md

```diff
@@ -62,8 +72,6 @@ rule = "github:typelevel/cats/v1.0.0"
 
 #### replace:
 
-⚠️ Experimental
```

Comment:
Why?

Reply:
Because it's a useful feature and any bugs should be fixed. Makes no sense to have a feature labeled "Experimental" for all eternity.

Reply:
Cool, I will update the release note.

Reply:
No need to mention this in the release notes, nothing has changed, this is just normal website cleanup